### PR TITLE
Validate BOOP

### PR DESCRIPTION
**Pull request by HasukiOfficial**
# Validate [BOOP](https://solscan.io/token/CCRJohp9bfQdhZCvCe7MzUQuXMJkKBi4XCjUq3A2YqN8)

## Attestations (Please provide links):
- Tweet from your Twitter Account attesting the Mint address, tagging [@JupiterExchange](https://twitter.com/JupiterExchange) and showing community support: 
- https://twitter.com/HasukiNFTs/status/1641761410299355137
- Coingecko/ CMC URL (If available): Not available.

## Validation (Please check off boxes):
- [x] The metadata provided in the PR matches what is on-chain (Mandatory)
- [x] Does not duplicate the symbol of another token on Jupiter's strict list (If not, review will be delayed)
- [ ] Is Listed on Coingecko / CMC (Optional, but helpful for reviewers)  

## Acknowledgement (Please check off boxes)
- [x] My change matches the format in the file (no spaces between fields).
- [x] My token is already live and trading on Jupiter.
- [x] !!! I read the README section on Community-Driven Validation and understand this PR will be only be reviewed when there is community support on Twitter.

### CODE REVIEW

**Comment by scastradio** [comment]
I can hereby attest that the provided information are accurate, and the token is valid.

Mint address: CCRJohp9bfQdhZCvCe7MzUQuXMJkKBi4XCjUq3A2YqN8
 Symbol: BOOP
 Token Decimals: 10
 Token icon: https://arweave.net/YTXQL4GzZTCijix2GEVv_nwNs4hlgAaqcimTT6e-H5Q
 On-chain metadata matches
 No duplication on Jupiter's strict list

File: validated-tokens.csv [hunk 1]
```diff
@@ -704,3 +704,4 @@ Nectar,NCTR,AgnHzGspNu7F3nFM4izuPt5g7m1URjVaTaFNgvqSXcjC,9,https://www.arweave.n
 BEAR,BEAR,9LmCL3nyvFG95cB9RpaszPTAwaQykVAHcSxQbJ6vNpAU,0,https://nova-host.nyc3.cdn.digitaloceanspaces.com/spl_tokens/9LmCL3nyvFG95cB9RpaszPTAwaQykVAHcSxQbJ6vNpAU.jpeg,true
 GM,GM,3acxNNmfdKKZj9i35P4VDBFm74Ufdt8ojKWceVGynwC5,4,https://arweave.net/jhNqKLDDC2ZtanzmFt-OZZJLFEYRpzus3rzhl7MGiOU,true
 Arbitrum (Portal from Arbitrum),ARB,8LH3QMo7xkMJx85Kg4pfiQY1g1ZgiVEe1KktSpaT89mP,8,https://assets.coingecko.com/coins/images/16547/small/arbitrium.png?1624418103,true
+BOOP,BOOP,CCRJohp9bfQdhZCvCe7MzUQuXMJkKBi4XCjUq3A2YqN8,10,https://arweave.net/YTXQL4GzZTCijix2GEVv_nwNs4hlgAaqcimTT6e-H5Q,true
```
Comment (Timoon21) [inline validated-tokens.csv:707]:
```suggestion
BOOP,BOOP,CCRJohp9bfQdhZCvCe7MzUQuXMJkKBi4XCjUq3A2YqN8,10,https://arweave.net/2ArDrNlyF34gjZ7UdXmMVl_ET9igdcyDiAY-VFsw3ao,true
```

**Review by Timoon21 — CHANGES_REQUESTED** [review]
LogoURI don't match on-chain metadata.

**Comment by Timoon21** [comment]
> I can hereby attest that the provided information are accurate, and the token is valid.
> 
> Mint address: CCRJohp9bfQdhZCvCe7MzUQuXMJkKBi4XCjUq3A2YqN8 Symbol: BOOP Token Decimals: 10 Token icon: https://arweave.net/YTXQL4GzZTCijix2GEVv_nwNs4hlgAaqcimTT6e-H5Q On-chain metadata matches No duplication on Jupiter's strict list

Please review carefully. This is not the token icon file.

**Comment by Timoon21** [comment]
> I can hereby attest that the provided information are accurate, and the token is valid.
> 
> Mint address: CCRJohp9bfQdhZCvCe7MzUQuXMJkKBi4XCjUq3A2YqN8 Symbol: BOOP Token Decimals: 10 Token icon: https://arweave.net/YTXQL4GzZTCijix2GEVv_nwNs4hlgAaqcimTT6e-H5Q On-chain metadata matches No duplication on Jupiter's strict list

Please review carefully. This is not the token icon file.

**Comment by HasukiOfficial** [comment]
Token icon file updated

**Review by Timoon21 — APPROVED** [review]
- [x] On-chain metadata matches the PR
- [x] Token address matches the tweet
- [x] No duplication on Jupiter's strict list
- [x] No duplication on CMC/CoinGecko
- [x] Community validation

**Comment by LJQZZZ123** [comment]
- [ ] On-chain metadata matches the PR
- [ ] Token address matches the tweet
- [ ] The same token symbol can be searched on CMC but seems unrelated
- [ ] No duplication on Jupiter's strict list
- [ ] Known by the community

> https://coinmarketcap.com/currencies/boop/

**Review by LJQZZZ123 — APPROVED** [review]

**Pull request merged**